sql/core/src/main/java/org/apache/spark/sql/execution/vectorized/WritableColumnVector.java

```diff
@@ -21,12 +21,14 @@
 
 import com.google.common.annotations.VisibleForTesting;
 
+import org.apache.orc.OrcProto;
```

Comment:
Is this used? Please run `dev/lint-java` and fix the errors.
```
[ERROR] src/main/java/org/apache/spark/sql/execution/vectorized/WritableColumnVector.java:[24,8] (imports) UnusedImports: Unused import - org.apache.orc.OrcProto.
```

Reply:
thanks